app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/LearningAudioFeedback.kt

```diff
@@ -0,0 +1,56 @@
+package com.github.HumanLearning2021.HumanLearningApp.view
+
+import android.content.Context
+import android.media.MediaPlayer
+import com.github.HumanLearning2021.HumanLearningApp.R
+
+class LearningAudioFeedback(private val applicationContext : Context) {
+    private lateinit var correctMp: MediaPlayer
+    private lateinit var incorrectMp: MediaPlayer
+
+    fun initMediaPlayers(){
+        correctMp = MediaPlayer.create(applicationContext, R.raw.bravo)
+        incorrectMp = MediaPlayer.create(applicationContext, R.raw.dommage)
+    }
+
+    fun releaseMediaPlayers(){
+        // we release when the user pauses the so that we don't eat up resources uselessly
+        correctMp.release()
+        incorrectMp.release()
+    }
```

Comment:
I think you forgot a word in the comment right?

Reply:
yup, was probably tired xD